src/components/button-toggle/README.md

```diff
@@ -0,0 +1,97 @@
+# MdButtonToggle
+
+`MdButtonToggle` is a group of buttons that can be toggled. There are two modes, `multiple` and
```

Comment:
You might want to take a step back and start with importing MD_BUTTON_TOGGLE_DIRECTIVES. Definitely mention that you need `MdUniqueSelectionDispatcher` and give an example of that.